docs/configuration/index-config.md

```diff
@@ -580,5 +580,5 @@ retention:
   - `years`, `year`, `y` -- a year is defined as `365.25 days`
 
 (2) `cutoff_reference` possible values:
-  - `publish_timestamp` will evaluate based on the timestamp the split was published at.
+  - `indexing_timestamp` will evaluate based on the timestamp the split was finalized at.
```

Comment:
we could be a bit clearer here. Something like 

"The retention policy will remove all splits with `formula`."

One thing is not clear too: we don't know what "split finalization" means. We should probably say something like "indexing_timestamp" corresopnds to the timestamp at which we stopped indexing document in the split.